README.md

```diff
@@ -1,135 +1,69 @@
+contributions in the form of [pull requests](https://github.com/open-ideas/IDEAS/pulls) are highly encouraged. Before submitting
+a pull request, please consult the the IDEAS
+[Style Guide and Conventions](https://github.com/open-ideas/IDEAS/wiki/Style%20Guide%20and%20GitHub%20Good%20Practice)
+to ensure consistency with the project's conventions.
+
+The IDEAS library was originally developed by KU Leuven and [3E](https://3e.eu), and is currently developed and maintained by
+the [Thermal Systems Simulation (The SySi)](http://the.sysi.be) research group of KU Leuven. 
+The library includes significant contributions by the [Building Physics and Sustainable Design Section](https://bwk.kuleuven.be/bwf)
+of KU Leuven, the [Building Physics Research Group](https://www.ugent.be/ea/architectuur/en/research/research-groups/building-physics)
+of UGent, [IBPSA project 1](https://ibpsa.github.io/project1/), [IEA EBC Annex 60](https://iea-annex60.org) and the
+[Electrical Energy Systems and Applications Section](https://www.esat.kuleuven.be/electa) of KU Leuven.
 
 ## Tool support
-Our goal is to provide support for Dymola and OpenModelica. Consequently, any tool that supports the full Modelica specification should be able to run our models. Feel free to file a bug report in case we do not adhere to the Modelica specification.
+IDEAS is fully compliant with the [Modelica Specification](https://specification.modelica.org/), and is therefore, in
+principle, compatible with any simulation tool that supports this standard, such as Dymola or OpenModelica. However, as
+IDEAS is primarily developed and maintained in Dymola, and the unit testing framework is exclusively implemented in Dymola,
+certain compatibility issues with alternative tools may arise. Users are encouraged to report any deviations from the Modelica
+specification or tool-specific issues so that these can be addressed in future updates.
 
-## Release notes
-[This is a link to detailed release notes.](https://github.com/open-ideas/IDEAS/blob/master/ReleaseNotes.md)
+## Unit testing
+Unit testing is performed using Dymola in combination with BuildingsPy. Automated unit tests are executed via GitHub Actions,
+utilising a self-hosted runner maintained by KU Leuven. Further information on the IDEAS unit testing framework is available
+on the [IDEAS Wiki](https://github.com/open-ideas/IDEAS/wiki).
 
-## Unit tests
-The library is unit tested using BuildingsPy. Automated unit tests are run on GitHub Actions with a self-hosted runner at KU Leuven.
+## Getting started
+Two tutorials are provided at <code>IDEAS.Examples.Tutorial</code>, serving as demonstrations cases of how the IDEAS library
+can be used.
+1. <code>IDEAS.Examples.Tutorial.SimpleHouse</code> contains examples with step-by-step instructions for how to build a system
+model for a simple house with a heating system, ventilation, and weather boundary conditions.
```

Comment:
Maybe emphasize that this one uses a very simple building model, not using the IDEAS building envelope component models.